rllab/algos/batch_polopt.py

```diff
@@ -107,6 +107,7 @@ def __init__(
     def start_worker(self):
         self.sampler.start_worker()
         if self.plot:
+            plotter.init_worker()
```

Comment:
This shouldn't be here. init_worker initializes process-global state and should only ever be called once in the entire program. Typically this is in run_experiment_lite() or the launcher script.

Reply:
This should be removed, as it is the responsibility (per the API) of the user to call `plotter.init_worker()` before calling `algo.train()`. I discuss possibility of changing the API in my comment below.